src/sentry/services/hybrid_cloud/user/impl.py

```diff
@@ -218,6 +218,18 @@ def verify_any_email(self, *, email: str) -> bool:
             return True
         return False
 
+    def verify_user_emails(
+        self, *, user_id_emails: List[Tuple[int, str]]
```

Comment:
I'm not certain that RPC requests will unpack into tuples. You might only be able to get `List`